integrations/docker/images/chip-build-nrf-platform/Dockerfile

```diff
@@ -30,6 +24,49 @@ RUN set -x \
     | tar jxvf -) \
     && : # last line
 
+# ==================================================
+# nRF5 SDK image (target: chip-build-nrf-platform)
+# ==================================================
+FROM common AS chip-build-nrf-platform
```

Comment:
I'm new to this syntax.  Where does this "FROM" directive end?  At the next "FROM"?

Reply:
yes